### PR TITLE
Create issues-workflow.md

### PR DESCRIPTION
**Pull request by pgebheim**
First cut of issues workflow descriptions

### CODE REVIEW

**Review by adrake33 — APPROVED** [review]
Lgtm

**Review by Chwy5 — APPROVED** [review]

**Review by ryanberckmans — APPROVED** [review]

File: issues-workflow.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- New Issues
+  - Where new external issues show up
+  - DIP will monitor this pipeline
+- Backlog
+  - This should be an ordered list of tasks per epic / release.
+- Blocked / Needs Information
+  - New issues that need exploration, or issues that were in progress but are now blocked by another task should go here. Feel free to move items in and out of this as need be!
+- In Progress
+  - Things which you are actively working on. If an item becomes blocked try to move it back to the Blocked / Needs Info pipeline. If you stop working on it, it goes back into the Backlog.
+- QA
```
Comment (robert-e-davidson3) [inline issues-workflow.md:58]: Why's this called "QA"?

File: issues-workflow.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - For integration work that needs approval from Product. Put the issue here and assign to the stakeholder that needs to check it out.
+- Closed
```
Comment (justinbarry) [inline issues-workflow.md:60]: This could also be situational but does closed == published or just in master?

**Review by bthaile — APPROVED** [review]
I got a bit lost in the “if this then that” logic. I’m sure I can get it when I either miss the process or it’s explained to me via voice.

**Review by matt-bullock — APPROVED** [review]

**Review by matt-bullock — APPROVED** [review]

**Pull request merged**